services/order.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None,
+) -> Order:
+    with transaction.atomic():
+        order = Order.objects.create(user=User.objects.get(username=username))

```

Comment:
Use get_user_model() instead of User. Check out the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency). Fix it everywhere)